tests/unit/SourceStorage/Snowflake/TableTest.php

```diff
@@ -23,6 +23,8 @@ public function testDefaultValues(): void
         self::assertInstanceOf(Storage\SourceInterface::class, $source);
         self::assertEquals('schema', $source->getSchema());
         self::assertEquals('table', $source->getTableName());
+        self::assertEquals([], $source->getQueryBindings());
+        self::assertEquals('"schema"."table"', $source->getFromStatement());
```

Comment:
Tohle se mi moc nelíbí - že se nějaký vlastně skoro "business" objekt plete do escapování DB. Potenciálně to bude problém pak pro Azure - kde se v MSSQL escapují identifiery pokud vím `[identifier]`... Nejspíš se pak nevyhneme nějaké další abstrakci :( Něco jako `new Identifier(new SchemaName('x'), new TableName('y'));`

Reply:
Ale ono je to vlastně zavřené v SNFLK namespacu a tím pádem AzureDB bude mít jiný... tak OK